core/extensions/layout/tabbar.ts

```diff
@@ -73,6 +78,22 @@ export class TabBar {
   }
 }
 
+class NewTabButton {
+  readonly dom: HTMLButtonElement;
+
+  constructor(private parent: LayoutView) {
+    this.dom = document.createElement("button");
+    this.dom.classList.add("new-tab-button");
```

Comment:
Because this button doesn't have text content, this should have an `aria-label` of something like "New Document" for screen readers

Reply:
good shout! https://github.com/mindofmatthew/text.management/pull/66/commits/338bbf02bacbb9b540080b03514d616e63996001